conf/parsers.conf

```diff
@@ -43,10 +43,11 @@
     Time_Key     time
     Time_Format  %Y-%m-%dT%H:%M:%S.%L
     Time_Keep    On
-    # Command      |  Decoder | Field | Optional Action
-    # =============|==================|=================
-    Decode_Field_As   escaped    log
-    Decode_Field_As   escaped    stream
+    # Command      |     Decoder   | Field | Optional Action
+    # =============|===============|=======|================
+    Decode_Field_As   escaped_utf8   log        do_next
+    Decode_Field_As   escaped        log
+    Decode_Field_As   escaped        stream
```

Comment:
why unescape stream?

Reply:
So far it's mainly std stream, as I can see from the previous comment: [Stream should be one of stdout|stderr](https://github.com/donbowman/fluent-bit/commit/4389effaab2e6ba299d2f010fd143e1036aa3d21)